instrumentation/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/OpenTelemetryAutoConfiguration.java

```diff
@@ -68,6 +73,31 @@ public SdkTracerProvider sdkTracerProvider(
           .build();
     }
 
+    @Bean
+    @ConditionalOnMissingBean
+    public SdkLoggerProvider sdkLoggerProvider(
+        ObjectProvider<List<LogRecordExporter>> loggerExportersProvider, Resource otelResource) {
+
+      SdkLoggerProviderBuilder loggerProviderBuilder = SdkLoggerProvider.builder();
+      loggerProviderBuilder.setResource(otelResource);
+
+      initializeGlobalLoggerProvider(loggerExportersProvider);
+
+      return loggerProviderBuilder.build();
+    }
+
+    private static void initializeGlobalLoggerProvider(
+        ObjectProvider<List<LogRecordExporter>> loggerExportersProvider) {
+      SdkLoggerProviderBuilder loggerProviderBuilder = SdkLoggerProvider.builder();
+      loggerExportersProvider.getIfAvailable(Collections::emptyList).stream()
+          .forEach(
+              loggerExporter ->
+                  loggerProviderBuilder.addLogRecordProcessor(
+                      SimpleLogRecordProcessor.create(loggerExporter)));
+      SdkLoggerProvider loggerProvider = loggerProviderBuilder.build();
+      GlobalLoggerProvider.set(loggerProvider);
```

Comment:
Note for the future (next release, actually): we'll have to set `GlobalOpenTelemetry` if we want appenders to work. Not that there's anything wrong with that, it's just that it's a behavioral change that we'll have to introduce.